__snapshots__/client/components/Sidebar/Sidebar.test.tsx.snap

```diff
@@ -2,41 +2,42 @@
 
 exports[`Test Sidebar Sidebar renders correctly 1`] = `
 <div
-  className="sc-fzpkJw fyrmec sidebar"
```

Comment:
oof, and this is why i pretty much hate snapshot tests now, Cypress tests are so much better